.ci/benchmarks.groovy

```diff
@@ -85,6 +96,14 @@ pipeline {
             }
           }
         }
+        success {
+          dir("${BASE_DIR}") {            
+            sendSlackReportSuccessMessage()
+          }
+        }
+        failure {
+          notifyBuildResult(slackComment: true)
```

Comment:
default behaviour won't notify slack messages for PRs, https://github.com/elastic/apm-pipeline-library/blob/0b1b3269e6a25bf50e5d78ed6102894c7f712fce/vars/notifyBuildResult.txt#L36

```suggestion
          notifyBuildResult(slackComment: true, slackNotify: true)
```